src/Utilities/unitTestingUtilities.js

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable react/prop-types */
+import { useSelector } from 'react-redux';
+import configureStore from 'redux-mock-store';
+/* eslint-disable-next-line */
+import { IntlProvider } from '@redhat-cloud-services/frontend-components-translations';
+
+export const initMocks = () => {
+  window.insights = {
+    chrome: {
+      appNavClick: () => {},
+      init: jest.fn(),
+      identifyApp: () => {},
+      navigation: () => {},
+      on: () => {
+        return () => {};
+      },
+      isBeta: () => true,
+      auth: {
+        getUser: () =>
+          new Promise((resolve) =>
+            resolve({
+              identity: {
+                user: {},
+              },
+            })
+          ),
+      },
+      getUserPermissions: () => Promise.resolve([]),
+    },
+    loadInventory: () =>
+      new Promise((resolve) => {
+        resolve({
+          inventoryConnector: () => {
+            const InventoryTable = ({ children }) => (
+              <div>A mock passed! {children} </div>
+            );
+            const InventoryDetailHead = ({ children }) => (
+              <div>A mock passed! {children} </div>
+            );
+            const AppInfo = ({ children }) => (
+              <div>A mock passed! {children} </div>
+            );
+            const DetailWrapper = ({ children }) => (
+              <div>A mock passed! {children} </div>
+            );
+
+            return {
+              InventoryTable,
+              InventoryDetailHead,
+              AppInfo,
+              DetailWrapper,
+            };
+          },
+          mergeWithEntities: () => {},
+          mergeWithDetail: () => {},
+        });
+      }),
+    loadRemediations: () =>
+      new Promise((resolve) => {
+        resolve({
+          openWizard: () =>
+            new Promise((resolve) => {
+              resolve(true);
+            }),
+        });
+      }),
+  };
+};
+
+export const mockStore = (initialState, mutatedState) => {
+  const customMiddleWare = () => (next) => (action) => {
+    useSelector.mockImplementation((callback) => {
+      return callback(mutatedState);
+    });
+    next(action);
+  };
+
+  // eslint-disable-next-line no-undef
+  const configuredMockStore = configureStore([customMiddleWare]);
+
+  return configuredMockStore(initialState);
+};
+
+export const mountWithIntl = (Component) => {
```

Comment:
As we are migrating away from Intl and you are using only text in this PR, I think we can remove this function.

Reply:
Will do 👍🏼 
Thank you @mkholjuraev !